app/views/groups/group_links/_namespace_group_link.html.erb

```diff
@@ -1,22 +1,21 @@
-<% member_source = membership_source(@namespace, namespace_group_link) %>
+<% namespace_group_link_source =
+  namespace_group_link_source(@namespace, namespace_group_link) %>
 <tr class="text-sm font-normal text-gray-500 dark:text-gray-400">
   <td class="px-6 py-4"><%= namespace_group_link.group.name %></td>
   <td class="px-6 py-4">
-    <% if !member_source.key?(:inherited_namespace_path) %>
-      <div class="text-sm font-normal text-gray-500 dark:text-gray-400">
-        <%= viral_tooltip(title: t(:"groups.members.index.shared_direct")) do %>
-          <%= member_source[:label] %>
-        <% end %>
-      </div>
-    <% else %>
-      <%= viral_tooltip(title: t(:"groups.members.index.shared_inherited")) do %>
-        <%= link_to member_source[:label],
-        member_source[:inherited_namespace_path],
+    <% if namespace_group_link_source.key?(:inherited_namespace_path) %>
+      <%= viral_tooltip(title: 'Inherited from') do %>
```

Comment:
**nitpick (blocking):** Can you please use a translated string for `Inherited from`

Reply:
Added in 038f17b9ee10a2609acd0c521ba6e415362e9ca2